device/dell/x86_64-dell_s6000_s1220-r0/installer.conf

```diff
@@ -1,4 +1,5 @@
 ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="processor.max_cstate=1 intel_idle.max_cstate=0"
+DOCKER_IN_RAM=y
```

Comment:
>DOCKER_IN_RAM [](start = 0, length = 13)

`DOCKER_BACKING_FILESYSTEM=ramfs`
alternatively the default is `DOCKER_BACKING_FILESYSTEM=ext4`

Reply:
docker_inram is better since you cannot really specify the docker backing filesystem as you like. you cannot really change ext4 to other filesystem other than ramfs.